### PR TITLE
README.md translated to English

### PR DESCRIPTION
**Pull request by peppelinux**
Closes https://github.com/teamdigitale/api-oas-checker/issues/21

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,104 +1,99 @@
...
 
 [![Join the #api channel](https://img.shields.io/badge/Slack-%23api-blue.svg?logo=slack)](https://developersitalia.slack.com/messages/CDKBYTG74)
 [![Get invited](https://slack.developers.italia.it/badge.svg)](https://slack.developers.italia.it/)
 [![API on forum.italia.it](https://img.shields.io/badge/Forum-interoperabilit%C3%A0-blue.svg)](https://forum.italia.it/c/piano-triennale/interoperabilita)
 
-Questo repository contiene un validatore in-browser che verifica
-alcune delle regole per le API REST indicate
-nel Modello di Interoperabilità.
+This repository contains an in-browser validator that verifies
+some of the rules for the REST APIs, as indicated in the Interoperability Model.
```
Comment (ioggstream) [inline README.md:8]:
```suggestion
some of the rules for the REST APIs, as indicated in the Interoperability Framework.
```

File: README.md [hunk 2]
```diff
@@ -1,104 +1,99 @@
...
-[qui](https://teamdigitale.github.io/api-oas-checker).
+The ready-to-use online application is available [here](https://teamdigitale.github.io/api-oas-checker).
 
-Rispetto ad altri validatori altrettanto buoni
-- eg. [Zally](https://github.com/zalando/zally) -
-Spectral può essere eseguito direttamente sul client (browser)
- e non richiede necessariamente l'utilizzo di un database.
+Compared to other equally good validators
+- eg. [Zally] (https://github.com/zalando/zally)
+- Spectral can be run directly on the client (browser)
```
Comment (ioggstream) [inline README.md:17]:
```suggestion
Spectral can run directly on the client (browser)
```

**Review by ioggstream — CHANGES_REQUESTED** [review]
@peppelinux Can you create a README.en.md file so that we retain Italian in the main file? Thanks!!!

File: README.md (hunk 1 above)
Comment (peppelinux) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (peppelinux) [reply README.md]: Done

**Comment by peppelinux** [comment]
> @peppelinux Can you create a README.en.md file so that we retain Italian in the main file? Thanks!!!

That's it, as requested

File: README.en.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+some of the rules for the REST APIs, as indicated in the Interoperability Model.
+
+The associated projects are indicated in the [API Starter Kit](https://github.com/teamdigitale/api-starter-kit).
+There's a Beta [github-action that uses these rules](https://github.com/teamdigitale/api-oas-checker-action).
+
+The ready-to-use online application is available [here](https://teamdigitale.github.io/api-oas-checker).
+
+Compared to other equally good validators
+- eg. [Zally] (https://github.com/zalando/zally)
+- Spectral can run directly on the client (browser)
```
Comment (ioggstream) [inline README.en.md:17]:
```suggestion
Spectral can run directly on the client (browser)
```

File: README.en.md [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+ and does not necessarily require the use of a database.
+
+## Content
+
+- A nodejs project that uses webpack to create a single-page application
+- A [rules /] (rules /) directory with the rules applied, which come
```
Comment (ioggstream) [inline README.en.md:23]:
```suggestion
- A [rules/](rules/) directory with the rules applied, which come
```

File: README.en.md [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+  then aggregate in the [spectral.yml] (spectral.yml) file.
```
Comment (ioggstream) [inline README.en.md:24]:
```suggestion
  then aggregate in the [spectral.yml](spectral.yml) file.
```

File: README.en.md [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+Jsonpath supports back-references,
+ see https://github.com/json-path/JsonPath/issues/287#issuecomment-265479196
+ 
+For more information on spectral rules see https://stoplight.io/p/docs/gh/stoplightio/spectral/docs/getting-started/rulesets.md
+
+## Thanks to
+
+- Paolo Falomo
+- Vincenzo Chianese
+
```
Comment (ioggstream) [inline README.en.md:99]:
```suggestion
- Giuseppe Di Marco
```

Comment (ioggstream) [reply README.en.md:99]: @peppelinux can you please add this to [README.md](README.md)?

**Review by ioggstream — CHANGES_REQUESTED** [review]
Almost done :)

File: README.en.md (hunk 4 above)
Comment (peppelinux) [reply README.en.md]: done

File: README.en.md (hunk 3 above)
Comment (peppelinux) [reply README.en.md]: Done

File: README.en.md (hunk 5 above)
Comment (peppelinux) [reply README.en.md]: Done

File: README.en.md (hunk 6 above)
Comment (peppelinux) [reply README.en.md]: Done

**Review by ioggstream — APPROVED** [review]
LGTM

**Pull request merged**